pkg/resource/aws/testdata/acc/aws_iam_group_policy/.driftignore

```diff
@@ -1,2 +1,2 @@
 *
-!aws_iam_group_policy
+!aws_iam_group
```

Comment:
I don't understand why changed the .driftignore of the `aws_iam_group_policy` test, can you explain?